WearOAuth/oauth-device-grant/src/main/res/layout/activity_auth.xml

```diff
@@ -29,7 +29,7 @@
             android:layout_width="wrap_content"
```

Comment:
Since we are using Compose, this xml can be removed alltogether!

Reply:
Done